climpred/metrics.py

```diff
@@ -126,14 +134,84 @@ def _mae(forecast, reference, dim='svd', comparison=None):
     return mae(forecast, reference, dim=dim)
 
 
-def _crps(forecast, reference, dim='svd', comparison=None):
+def _brier_score(forecast, reference, **kwargs):
+    """Calculate Brier score for forecasts on binary reference.
+
+    ..math:
+        BS(f, o) = (f - o)^2
+
+    Args:
+        * forecast (xr.object)
+        * reference (xr.object)
+        * func (function): function to be applied to reference and forecasts
+                           and then mean('member') to get forecasts and
+                           reference in interval [0,1]  (required)
+
```

Comment:
Add `**kwargs` to the Args here with an explanation of what they can be.

Reply:
> I can't test `brier` because I don't understand yet what kind of function I am supposed to put in.

see the example provided now